library/LuaApi.cpp

```diff
@@ -2373,7 +2373,7 @@ static int internal_setAddress(lua_State *L)
 
     // Print via printerr, so that it is definitely logged to stderr.log.
     uintptr_t iaddr = addr - Core::getInstance().vinfo->getRebaseDelta();
-    fprintf(stderr, "Setting global '%s' to %x (%x)\n", name.c_str(), addr, iaddr);
+    fprintf(stderr, "Setting global '%s' to %zu (%zu)\n", name.c_str(), addr, iaddr);
```

Comment:
MSVC complains when using `%p` for `uintptr_t` types, which makes sense as `uintptr_t` is defined to be an unsigned integer type capable of storing a pointer value, not a pointer type _per se_.